source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst

```diff
@@ -0,0 +1,103 @@
+.. _increase-virtual-disk-size:
+
+Increase Clear Linux image virtual disk size
+############################################
+
+Pre-built |CLOSIA| images come in different sizes, ranging anywhere
+from 300M to 20G.  This section shows you how to increase the size of your 
+pre-built |CL| image if you need more capacity.
+
+Determine the pre-built image size
+**********************************
+
+There are two methods you can use to find the virtual disk size of your 
+pre-built |CL| image.
+
+The first method is to check the image's config JSON file located in the 
+`releases`_ repository.  For example, to find the size of the 
+Hyper-V image version number 20450, you can follow these steps:
+
+#.	Go to the `releases`_ repository.
+#.	Drill down into the `20450 > clear > config > image` directory.
+#.	Open the :file:`hyperv-config.json` file.  
+#.	Locate the `PartitionLayout` key like this example.  This one shows an 
+	512MB for the EFI partition, 32MB for the swap partition, and 8GB for the 
+	root partition.    
+
+	.. code-block:: console
+
+	   "PartitionLayout" : [ { "disk" : "hyperv.img",
+	                            "partition" : 1,
+	                            "size" : "512M",
+	                            "type" : "EFI" },
+	                          { "disk" : "hyperv.img",
+	                            "partition" : 2,
+	                            "size" : "32M",
+	                            "type" : "swap" },
+	                          { "disk" : "hyperv.img",
+	                            "partition" : 3,
+	                            "size" : "8G",
+	                            "type" : "linux" } ],
+
+The second method is to simply boot up your :abbr:`VM (Virtual Machine)` and 
+execute the :command:`lsblk` command.  Here's a sample output:
+
+	.. code-block:: console
+
+		# lsblk
+
+		NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+		sda      8:0   0    8.5G  0 disk
+		├─sdd1   8:1   0    512M  0 part 
+		├─sdd2   8:2   0     32M  0 part [SWAP]
+		└─sdd3   8:3   0      8G  0 part /
+
+As you can see, both methods show the pre-built Hyper-V image is about 8.5GB. 
+
+Increase the root partition size
+********************************
+
+To increase the image size of a pre-built image, follow these steps:
+
+#.	Shutdown your VM, if it's running.  
+#.	Use the appropriate hypervisor tool to increase the virtual disk size of 
```

Comment:
Can you give an example of what type of tool you need to use for this?

Reply:
You would use the tool for your hypervisor.  I'm happy to add step-by-step for various VMs if you think this is a must-have.  I was trying to focus on the Clear aspect.